### ISSUE TITLE
Cannot deploy flex services from master

### ISSUE THREAD

**Issue by Hexcles** [web-platform-tests/wpt.fyi#1696]
This is different from #1652 . I'm seeing the following error both locally and on Travis:

> ERROR: (gcloud.app.deploy) Error Response: [13] Flex operation projects/wptdashboard-staging/regions/us-east4/operations/5bede3f0-8a91-4253-b95f-7352f681e788 error [INTERNAL]: An internal error occurred while processing task /appengine-flex-v1/insert_flex_deployment/flex_create_resources>2019-12-04T16:36:22.013Z30870.oj.3: Deployment Manager operation wptdashboard-staging/operation-1575477382662-598e369bec967-11b947ad-be69a90b errors: [code: "RESOURCE_ERROR"
> location: "/deployments/aef-searchcache-74e8154/resources/aef-searchcache-74e8154-00it"
message: "{\"ResourceType\":\"compute.v1.instanceTemplate\",\"ResourceErrorCode\":\"400\",\"ResourceError> Message\":{\"code\":400,\"errors\":[{\"domain\":\"global\",\"message\":\"Invalid value for field \'resource.properties.labels\': \'\'. Label value \'Infinity\' violates format constraints. The value can only contain lowercase letters, numeric characters, underscores and dashes. The value can be at most 63 characters long. International characters are allowed.\",\"reason\":\"invalid\"}],\"message\":\"Invalid value for field \'resource.properties.labels\': \'\'. Label value \'Infinity\' violates format constraints. The value can only contain lowercase letters, numeric characters, underscores and dashes. The value can be at most 63 characters long. International characters are allowed.\",\"statusMessage\":\"Bad Request\",\"requestPath\":\"https://compute.googleapis.com/compute/v1/projects/wptdashboard-staging/global/instanceTemplates\",\"httpMethod\":\"POST\"}}"

**Comment by Hexcles** [comment]
Found this: https://b.corp.google.com/issues/145383056

Tl;dr: this error happened at commit 74e8154. We use the short SHA as the version name. And this particular short SHA is interpreted as a float (scientific notation) by gcloud... Theoretically the error should go away in the next deployment.

**Comment by Hexcles** [comment]
Yup https://travis-ci.com/web-platform-tests/wpt.fyi/builds/139372766
